### PR TITLE
Task dev 11

### PR DESCRIPTION
**Pull request by ineskavalenka**

### CODE REVIEW

File: DEV-11/ru_en_4.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+фонд-Fund
+союз-union
```
Comment (AlexanderSoroka) [inline DEV-11/ru_en_4.txt:2]: What? it's not transliteration

File: DEV-11/src/EntryPoint.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+public class EntryPoint {
+  final static int MAX_SYMBOL_LENGTH_RU = 4;
+  final static int MAX_SYMBOL_LENGTH_EN = 4;
+
+  public static void main(String[] args) {
+    FileReader fileReader = new FileReader();
+    try {
+      HashMap[] ruEn = new HashMap[MAX_SYMBOL_LENGTH_RU+1];
+      for(int i=1; i<=MAX_SYMBOL_LENGTH_RU; i++) {
```
Comment (AlexanderSoroka) [inline DEV-11/src/EntryPoint.java]: codestyle violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]
Split transliteration rule to 8(?) dictionary files id quite bad idea. Please, rework it in order to find more elegant solution - at least put everything to one dictionary file

File: DEV-11/ru_en_4.txt (hunk 1 above)
Comment (ineskavalenka) [reply DEV-11/ru_en_4.txt:2]: it was in the task

**Pull request closed without merging**